packages/documentation/src/content/docs/integration/playground/overview.mdx

```diff
@@ -215,21 +225,22 @@ pnpm localenv:compose down --volumes --rmi all
 
 The following are the most commonly used commands:
 
-| Command                                        | Description                                                       |
-| ---------------------------------------------- | ----------------------------------------------------------------- |
-| pnpm localenv:compose config                   | Show all merged config (with TigerBeetle)                         |
-| pnpm localenv:compose up                       | Start (with TigerBeetle)                                          |
-| pnpm localenv:compose up -d                    | Start (with TigerBeetle) detached                                 |
-| pnpm localenv:compose down                     | Down (with TigerBeetle)                                           |
-| pnpm localenv:compose down --volumes           | Down and kill volumes (with TigerBeetle)                          |
-| pnpm localenv:compose down --volumes --rmi all | Down, kill volumes (with TigerBeetle) and images                  |
-| pnpm localenv:compose:psql config              | Show all merged config (with PostgreSQL)                          |
-| pnpm localenv:compose build                    | Build all the containers (with TigerBeetle)                       |
-| pnpm localenv:compose:psql up                  | Start (with PostgreSQL)                                           |
-| pnpm localenv:compose:psql down                | Down (with PostgreSQL)                                            |
-| pnpm localenv:compose:psql down --volumes      | Down (with PostgreSQL) and kill volumes                           |
-| pnpm localenv:compose:psql build               | Build all the containers (with PostgreSQL)                        |
-| pnpm localenv:compose:adminauth up             | Start with local admin auth enabled (this is disabled by default) |
+| Description                                                       | Command                                          |
```

Comment:
It is included in the diagram, but I think we should have a text explanation that Tigerbeetle enabled means it will be enabled for the first node (cloud nine wallet) only. The second instance (happy life bank) will always run PSQL as its accounting